### ISSUE TITLE
Plan B to Heroku

### ISSUE THREAD

**Issue by Chapagain43** [timoschlueter/nightscout-librelink-up#75]
If anyone is still looking for Plan B to Heroku, I have been using Azure for a little while now since it is back and both the NightScout website and this wonderful write app from Libre to NightScout works seamlessly.

Follow the same steps as migrating nightscout website from the link below and replace where necessary with librelink-up configurations
https://nightscout.github.io/vendors/azure/migrate/

**Comment by Chapagain43** [comment]
Getting this error anyone know what I need to do?

2022-11-07T21:31:31.511Z INFO - Logging is not enabled for this container.
Please use https://aka.ms/linux-diagnostics to enable logging to see container logs here.
2022-11-07T21:31:33.652Z INFO - Initiating warmup request to container libre2nightscout_0_9ad090eb for site libre2nightscout
2022-11-07T21:31:54.087Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 20.4351059 sec
2022-11-07T21:32:09.210Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 35.5578431 sec
2022-11-07T21:32:24.374Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 50.7215846 sec
2022-11-07T21:32:41.141Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 67.4891618 sec
2022-11-07T21:32:58.170Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 84.5175334 sec
2022-11-07T21:33:15.160Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 101.5080301 sec
2022-11-07T21:33:31.143Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 117.4907699 sec
2022-11-07T21:33:47.214Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 133.5618766 sec
2022-11-07T21:34:02.297Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 148.6453172 sec
2022-11-07T21:34:17.397Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 163.7447444 sec
2022-11-07T21:34:32.489Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 178.8371351 sec
2022-11-07T21:34:47.582Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 193.9296813 sec
2022-11-07T21:35:02.777Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 209.1247432 sec
2022-11-07T21:35:18.352Z INFO - Waiting for response to warmup request for container libre2nightscout_0_9ad090eb. Elapsed time = 224.6996132 sec
2022-11-07T21:35:24.762Z ERROR - Container libre2nightscout_0_9ad090eb for site libre2nightscout did not start within expected time limit. Elapsed time = 231.1101127 sec
2022-11-07T21:35:25.207Z ERROR - Container libre2nightscout_0_9ad090eb didn't respond to HTTP pings on port: 80, failing site start. See container logs for debugging.
2022-11-07T21:35:25.709Z INFO - Stopping site libre2nightscout because it failed during startup.

**Comment by durgok** [comment]
Hi were you successfull with this? I understand how to get the webpage via Azure up but not sure how to make Azure talk to Libre3 ? This code do not have easy option that points to deploy to Azure similar to drploy to Heroku. Can you help?

**Comment by Chapagain43** [comment]
Hi @durgok, Yes I was able to keep this running by deleting the webapp and starting fresh in Azure. 
When setting up use the repository instead of nightscout
And similar to nightscout add relevant username and password in configuration

**Comment by durgok** [comment]
@Chapagain43 Thanks, but I went with paid service nsdbe10. Heroku shut overnight because I used all the dynos in the month’s quota and I needed something immediately as my daughter was looping. So opted for the service, they charge me ~$5 every month, and it was so seamless service. And they help manage all the other stuff like upgrades/DB size/security etc., Since I was anyhow paying the same for Heroku, just switched over. Appreciate you getting back though. Am sure this response will be useful to others who come searching for Azure option.
